### DISCUSSION TITLE
请问能添加一个回到旧版首页的功能插件吗？

### DISCUSSION THREAD

**Discussion by AWES25** [the1812/Bilibili-Evolved#4517] [功能建议 / Ideas]
请问能添加一个回到旧版首页的功能插件吗？现在的B站没法回到旧版首页了，新版没有分区条，太难用了

**Comment by the1812** [comment]
https://github.com/MotooriKashin/Bilibili-Old

**Comment by Geeyun-JY3** [comment]
目前我这边实测 https://greasyfork.org/scripts/455571 这个脚本还能回到旧版首页（没[楼上那个](https://github.com/the1812/Bilibili-Evolved/discussions/4517#discussioncomment-7522848)那么旧），也不知道还能撑多久就是了。
